.travis.yml

```diff
@@ -3,5 +3,6 @@ git:
   depth: 2
 language: node_js
 node_js:
+  - 'iojs'
```

Comment:
The reason why `0.12` and `0.10` are quoted is that they are regarded as `Number` without quotes. I think the current style is consistent in the way it does not use explicit quotes as possible.

But, certainly, I'm too opinionated. I didn't know [the Travis CI doc uses that style](http://docs.travis-ci.com/user/languages/javascript-with-nodejs/). OK, now I don't care about quotes.